components/list/test/list-item-expand-collapse-mixin.test.js

```diff
@@ -18,7 +18,7 @@ const tag = defineCE(
 
 describe('ListItemExpandCollapseMixin', () => {
 	it('Sets expandable to false when no key is given', async() => {
```

Comment:
Oh, looks like this test is actually testing behaviour for the absence of a key. Lol.